packages/amplify-headless-interface/src/interface/auth/add.ts

```diff
@@ -187,6 +184,32 @@ export interface CognitoSocialProviderConfiguration {
   clientSecret: string;
 }
 
+interface SignInWithAppleSocialProviderConfig {
```

Comment:
Add a description comment to this interface and `SocialProviderConfig` above

Reply:
commenting complete